custom/icds_reports/data_pull/data_pulls.py

```diff
@@ -0,0 +1,43 @@
+from custom.icds_reports.data_pull.exceptions import UnboundDataPull
+
+
+class BaseDataPull:
+    slug = ""
+    queries = []
```

Comment:
might be safer to make this `None` since I think the list could accidentally get modified globally

Reply:
Might be worth documenting the expected type of what goes in this field too

Reply:
https://github.com/dimagi/commcare-hq/pull/26304/commits/7462b24f046acaed74e90a3f9a650df0b5781d44